src/vcpkg/vcpkgcmdarguments.cpp

```diff
@@ -925,6 +930,8 @@ namespace vcpkg
 
     constexpr StringLiteral VcpkgCmdArguments::TRIPLET_ENV;
     constexpr StringLiteral VcpkgCmdArguments::TRIPLET_ARG;
+    constexpr StringLiteral VcpkgCmdArguments::HOST_TRIPLET_ENV;
+    constexpr StringLiteral VcpkgCmdArguments::HOST_TRIPLET_ARG;
```

Comment:
now that we've removed VS2015 from our support matrix, are these still necessary?

Reply:
I _thought_ these were for GCC. Regardless, I think it's safest to follow the existing pattern and remove them all in a separate PR.